config/migrations/2023/20230116120500-add-status-concepts.ttl

```diff
@@ -0,0 +1,24 @@
+
+
+<http://lblod.data.gift/concept-schemes/effb1058-d469-4064-a7a6-9d98ba081962> a <http://www.w3.org/2004/02/skos/core#ConceptScheme>;
+  <http://www.w3.org/2004/02/skos/core#prefLabel> "Statuses of public services";
```

Comment:
It's not a status of the public service, it kinda summarizes what the state of the linked conceptual service has been.

`Statuses of linked conceptual public service`

Reply:
Second thought. We want to say two things.
We want to inform the user some changes happened on the concept and they might want to review their status.
We also want to say what happened to the status; updated or deleted.